### ISSUE TITLE
Use <nav> in the docs page for table of contents, breadcrumbs (prev/next) etc

### ISSUE THREAD

**Issue by sarahboyce** [django/djangoproject.com#1956]
https://github.com/django/djangoproject.com/blob/main/docs/templates/docs/doc.html

Using `<nav>` for items such as the table of contents improves the experience for screenreader users. The items should be labeled correctly (associated to the labels)

The language and docs version picker may also want to be in a `<nav>`

[This comment](https://github.com/django/djangoproject.com/issues/1411#issuecomment-2676972984) has an outline of what I have in mind

**Comment by kedar49** [comment]
can I work on this issue ?

**Comment by bmispelon** [comment]
Had to revert the commit, so this is still open

**Comment by bmispelon** [comment]
Fix reapplied in 1ee5166d50806b9da29f64c440c87beec5b6516d
